.github/workflows/test-application.yaml

```diff
@@ -104,6 +96,12 @@ jobs:
                   restore-keys: |
                       ${{ runner.os }}-composer-
 
+            - name: Composer require
+              if: ${{ matrix.symfony-version == '^4.4' }}
+              # symfony 4.4 not compatible with doctrine/annotations 2.0: https://github.com/symfony/symfony/issues/48717#issuecomment-1359164836
+              run: |
+                  composer require --no-update doctrine/annotations:^1.10
+
```

Comment:
Required for: Error: Call to undefined method Doctrine\Common\Annotations\AnnotationRegistry::registerLoader() in PHP 7.3 with Symfony 4.4